src/newstuff/rtfPopup.cpp

```diff
@@ -8,18 +8,18 @@
 
 #include "../interop/QtDotNetConverters.h"
 
-RtfPopup::RtfPopup(System::String^ rtfText, QWidget* parent, Qt::WindowFlags f) : QDialog(parent, f)
+RtfPopup::RtfPopup(System::String^ rtfText, QWidget* parent, Qt::WindowFlags f) : QMainWindow(nullptr, f)
```

Comment:
Why do we still take the parent argument if we're always using nullptr?

Reply:
I kept the parameter and use it but now I pass `nullptr` explicitly when needed so that we can still pass other parents.